### DISCUSSION TITLE
Animations not working

### DISCUSSION THREAD

**Discussion by amrap030** [tailwindlabs/tailwindcss#2201] [Help]
Hello, I installed the latest tailwindcss v1.7.2, but for some reasons animations are not working for me. I want to use it in my Vue.js project but don't know why it is not working. It would be great if someone can help me.

**Comment by simonswiss** [comment]
Hi 👋

Do you have a code snippet to share so we can help troubleshoot? The more information you share, the easier it will be to provide help! 👍

**Reply by amrap030** [reply]
I just updated tailwind via `npm install tailwindcss@^1.0 --save-dev` as I had installed it already. Now I thought it should add the animation utilities and work out of the box or do I have to adjust something? No matter where I put for exampe `animate-pulse` it doesn't work.

**Reply by amrap030** [reply]
It is still not working for me. Could somebody please help?

**Comment by amrap030** [comment]
I noticed that the classes are working when I build the app for production. So in development the classes are not available. Do I have to rebuild the tailwind classes for development with a special command?

**Reply by simonswiss** [reply]
Sounds like you're perhaps running Purge CSS on your initial run of Taiwind, and are not watching for changes in your HTML, so when you add animation classes, they are not present in the bundled output.

You probably need to move Purge CSS out of development mode, or have a task that watches for changes in your HTML.

Are you able to create a little public GitHub repo with your setup so I can have a look?

Alternatively, you could try clone the [Tailwind Playground](https://github.com/tailwindlabs/tailwindcss-playground) repo, and use the animation classes within that environment.

**Comment by amrap030** [comment]
Hello Simon,

this is my public repo: https://github.com/amrap030/tailwindanimations

First npm install and then npm run serve. You can login to the application with simonswiss@test.com and password123.

**Reply by adamwathan** [reply]
Just installed and set it up and the animations are working as expected, I can see the header with "Training" fading in and out.

**Reply by amrap030** [reply]
@adamwathan Hmm strange, when I clone this repository into a new folder and do a npm install & npm run serve it works but in my original project it doen't work. I already deleted the node_modules folder and did npm install in my original project but it still doesn't work.

**Reply by simonswiss** [reply]
Definitely working for me too!

![Large GIF (774x478)](https://user-images.githubusercontent.com/485747/91371672-f562a100-e854-11ea-87e1-c649b81e116c.gif)

Does your terminal or browser console give you any errors you can use to troubleshoot?

Dashboard UI looking good by the way ✨
